### PR TITLE
Add first-test-go

### PR DESCRIPTION
**Pull request by jonatasbaldin**
Hi there!

I'm adding a sample test code and instructions with Golang.

### CODE REVIEW

File: first-test-go/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# First Test with Golang
+This is an example repository with a simple test for demonstrating [Testcontainers Cloud](https://testcontainers.cloud).
+
+Close the repository:
```
Comment (clive-jevons) [inline first-test-go/README.md]: `s/Close/Clone/`? 😬

Comment (jonatasbaldin) [reply first-test-go/README.md]: Fixed, thanks for spotting!

File: first-test-go/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+=== RUN   TestWithRedis
+2022/04/19 15:40:36 Starting container id: 396fd04ae829 image: docker.io/testcontainers/ryuk:0.3.3
+2022/04/19 15:40:37 Waiting for container id 396fd04ae829 image: docker.io/testcontainers/ryuk:0.3.3
+2022/04/19 15:40:37 Container is ready id: 396fd04ae829 image: docker.io/testcontainers/ryuk:0.3.3
+2022/04/19 15:40:37 Starting container id: 37b6361734ff image: redis:latest
+2022/04/19 15:40:37 Waiting for container id 37b6361734ff image: redis:latest
+2022/04/19 15:40:38 Container is ready id: 37b6361734ff image: redis:latest
+    main_test.go:40: M1:C 19 Apr 2022 13:40:38.036 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
+        n1:C 19 Apr 2022 13:40:38.036 # Redis version=6.2.6, bits=64, commit=00000000, modified=0, pid=1, just started
+        �1:C 19 Apr 2022 13:40:38.036 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
```
Comment (kiview) [inline first-test-go/README.md]: Somet6ing interesting happened here with the encoding 😅

File: first-test-go/main.go [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package main
+
+func main() {
```
Comment (kiview) [inline first-test-go/main.go]: Is this file even needed?

**Review by clive-jevons — APPROVED** [review]
lgtm

File: first-test-go/main.go (hunk 3 above)
Comment (jonatasbaldin) [reply first-test-go/main.go]: not really, my IDE made it up :( removed

File: first-test-go/main_test.go [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	"testing"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+func TestWithRedis(t *testing.T) {
+	ctx := context.Background()
+	req := testcontainers.ContainerRequest{
+		Image:        "redis:latest",
```
Comment (bsideup) [inline first-test-go/main_test.go]: Let's avoid :latest

**Review by kiview — APPROVED** [review]

**Pull request merged**